### PR TITLE
VideoJS Player

### PR DESCRIPTION
**Pull request by aprigge**
Connected to [APPS-1056](https://jira.library.ucla.edu/browse/APPS-1056)

Acceptance Criteria:
- [x] This player is working on safari, chrome, firefox, etc on IOS and other operating systems

### CODE REVIEW

File: src/components/DLViewer.vue [hunk 1]
```diff
@@ -1,28 +1,27 @@
 <template>
   <div class="dl-viewer">
-    <MejsPlayer v-if="isVideo" :src="src" />
+    <VideoJS v-if="isVideo" :options="videoOptions" />
     <UniversalViewer
       v-else
       :iiif_manifest="iiif_manifest"
       :iiif_manifest_url="iiif_manifest_url"
       :uv_config="parseConfig"
-      :media="media"
```
Comment (pghorpade) [inline src/components/DLViewer.vue]: It needs this prop.

File: src/components/VideoJS.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="video-js-container">
+    <div class="video-player">
+      <video
+        ref="videoPlayer"
+        class="video-js vjs-big-play-centered"
```
Comment (pghorpade) [inline src/components/VideoJS.vue:6]: vjs-big-play-centered good find

**Review by pghorpade — CHANGES_REQUESTED** [review]
Good work, just one change requested.

File: src/components/DLViewer.vue (hunk 1 above)
Comment (aprigge) [reply src/components/DLViewer.vue]: oops, must have taken out resolving. conflicts.

**Review by pghorpade — APPROVED** [review]
💯

**Pull request merged**